src/app/components/OptimoPromos/Link/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { node, string, shape } from 'prop-types';
+import useCombinedClickTrackerHandler from '#containers/StoryPromo/useCombinedClickTrackerHandler';
+import StyledLink from './index.styles';
+
+const Link = ({ className, children, toLink, a11yId, eventTrackingData }) => {
+  const handleClickTracking = useCombinedClickTrackerHandler(eventTrackingData);
+  return (
+    <StyledLink
+      className={className}
+      href={toLink}
+      aria-labelledby={a11yId}
```

Comment:
I would have to dig through the code to find out what `a11yId` is for. Maybe `ariaLabelledBy` is better?

Reply:
Agree, can't think of a better name convention for this one. Although this value gets used as the key in the map as well, I think that's minor. `uniqueId` maybe? but might not be enough descriptive... For now I change this to `ariaLabelledBy`.